app/jobs/get_next_ai_message_job.rb

```diff
@@ -0,0 +1,45 @@
+class GetNextAiMessageJob < ApplicationJob
+  def perform(conversation_id)
+    puts "GetNextAiMessageJob.perform(#{conversation_id})"
+
+    @conversation = Conversation.find conversation_id
+    messages = @conversation.messages.order(:id).map(&:for_openai)
+    @new_message = @conversation.messages.create! role: :assistant, content_text: "", assistant: @conversation.assistant
+    @new_message.broadcast_append_to @conversation
+
+    response = OpenAI::Client.new(
+      access_token: @conversation.user.openai_key,
+    ).chat(
+      parameters: {
+        model: "gpt-3.5-turbo",
+        messages: messages,
+        temperature: 0.8,
+        stream: build_response_handler,
+        n: 1
+      }
+    )
+
+    @conversation.broadcast_refresh
+    puts "Finished GetNextAiMessageJob.perform(#{conversation_id})"
+
+  rescue => e
+    puts "Error in GetNextAiMessageJob: #{e.inspect}"
+    puts e.backtrace
+  end
```

Comment:
SolidQueue has some error handling stuff, but I haven't spent any time digging into it yet.